ISSUE_GUIDELINES.md

```diff
@@ -19,14 +20,16 @@ ___
 <!-- tocstop -->
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project